examples/prediction_upload/geospatial_predictions.ipynb

```diff
@@ -67,13 +67,10 @@
       "cell_type": "code",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.      global_keys=global_key, # A list of data rows or data row ids
</pre><p><code>global_keys=[global_key]</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1006/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>